src/openrct2/ride/RideGroupManager.h

```diff
@@ -40,3 +41,8 @@ class RideGroupManager
     static int32_t VehiclePreferenceCompare(const uint8_t rideType, const char * a, const char * b);
     static bool RideTypeIsIndependent(const uint8_t rideType);
 };
+
+enum RideGroupFlags : uint8_t
+{
+    RIDE_GROUP_FLAG_ALLOW_DOORS_ON_TRACK = 0b00000001,
+};
```

Comment:
Curious, which editor do you use? `.editorconfig` has a rule that adds a new line before the end of file.